.github/workflows/devcontainer-build.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v4
         with:
-          context: '.devcontainer'
+          context: 'helm-validate/.devcontainer'
           platforms: linux/amd64,linux/arm64
           push: true
           tags: ghcr.io/joaquinrz/helmv-devcontainer:latest
```

Comment:
we need to remove joaquinrz and friendly-fiesta and me sure that the workflows are building and pushing the images to retaildevcrews org

Reply:
Updated to pushing to retaildevcrews org